README.md

```diff
@@ -18,60 +18,46 @@ We use `main` branch as the develop branch while setting the default branch as t
-| AAD               | [spring-cloud-azure-starter-active-directory:4.3.0]          | [aad-resource-server-by-filter](aad/spring-cloud-azure-starter-active-directory/aad-resource-server-by-filter)                                                                                                                                                 |
-| AAD               | [spring-cloud-azure-starter-active-directory:4.3.0]          | [web-client-access-resource-server](aad/spring-cloud-azure-starter-active-directory/web-client-access-resource-server)                                                                                                                                         |
-| AAD               | [spring-cloud-azure-starter-active-directory:4.3.0]          | [aad-webapp-resource-server](aad/spring-cloud-azure-starter-active-directory/aad-web-application-and-resource-server)                                                                                                                                          |
-| App Configuration | [azure-spring-cloud-appconfiguration-config:2.8.0]           | [azure-spring-cloud-appconfiguration-config-sample](appconfiguration/azure-spring-cloud-appconfiguration-config/azure-spring-cloud-appconfiguration-config-sample)                                                                                             |
-| App Configuration | [azure-spring-cloud-appconfiguration-config:2.8.0]           | [azure-spring-cloud-appconfiguration-config-convert-sample-complete](appconfiguration/azure-spring-cloud-appconfiguration-config/azure-spring-cloud-appconfiguration-config-convert-sample/azure-spring-cloud-appconfiguration-config-convert-sample-complete) |
-| App Configuration | [azure-spring-cloud-appconfiguration-config:2.8.0]           | [azure-spring-cloud-appconfiguration-config-convert-sample-initial](appconfiguration/azure-spring-cloud-appconfiguration-config/azure-spring-cloud-appconfiguration-config-convert-sample/azure-spring-cloud-appconfiguration-config-convert-sample-initial)   |
-| App Configuration | [azure-spring-cloud-feature-management:2.7.0]                | [azure-spring-cloud-feature-management-sample](appconfiguration/azure-spring-cloud-feature-management/azure-spring-cloud-feature-management-sample)                                                                                                            |
-| App Configuration | [azure-spring-cloud-feature-management-web:2.7.0]            | [azure-spring-cloud-feature-management-web-sample](appconfiguration/azure-spring-cloud-feature-management-web/azure-spring-cloud-feature-management-web-sample)                                                                                                |
```

Comment:
It's weird that delete it in README but keep the sample code still exists in this branch.

Reply:
The readme file will keep syncing with the root pom file, some projects still exist but they are not included in the root pom file. The SDK repo used the same policy for spring-data starter.